### PR TITLE
Update API response

### LINKED ISSUE #19: Changes in the API response
**This is not an issue, but a feature 😄** 

I made a breaking change in the API response, the new response schema looks thus: 
```javascript
{
  fid: 1420775,
  countryCode: "US",
  date: "2021-11-07T20:30:00.000Z",
  league: "Major League Soccer",
  homeTeam: "Toronto FC",
  awayTeam: "DC United",
  odd: 1.53, // not always available 
  tip: "Over 2.5",
  score: "1-3", // mostly available after the match
  outcome: "WON"  // could be "LOST"
  isLeaguePerformant: "false" // will be removed later
},
```

Please expand the table column to match the example below.
Time | Country | League | Match | Tip | Odd | Score | Outcome | 
--- |--- |--- |--- |--- |--- |--- |--- |
20:30 | 🇺🇸 | Major League Soccer | Toronto FC vs DC United | Over 2.5 | 1.53 | 1-3 | Won | 

Hints:
1. You can use the `countryCode` to generate the country flags. Pay attention to country codes that have `-` (eg: GB-EN), this shouldn't stop you from completing this ticket but see if you can find a way around them.
2. It would be nice to be able to easily differentiate between wins and loses by colour.

### PR DESCRIPTION
**Pull request by Moween**
This feature updates table data with updated API response

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,5 +44,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "react-flags": "^0.1.18"
```
Comment (ebukaume) [inline package.json:49]: Why is this in dev dep?

File: src/components/TableData.jsx [hunk 2]
```diff
@@ -22,8 +33,11 @@ const TableData = ({ match }) => {
         </Box>
       </td>
       <td>
-        <div className="team">{homeTeam}</div>
-        <div className="team">{awayTeam}</div>
+        <span>{countryCode}</span>
+      </td>
+      <td>
+        <span className="team">{homeTeam}</span> vs{' '}
```
Comment (ebukaume) [inline src/components/TableData.jsx:39]: Doesn't this `vs` need its own span?

**Review by ebukaume — COMMENTED** [review]
Nice work! Just a few comments

File: package.json (hunk 1 above)
Comment (Moween) [reply package.json:49]: I really don't know she found herself there. But I  have removed her already.

**Pull request merged**